src/main/java/org/sagebionetworks/bridge/exporter/util/BridgeExporterUtil.java

```diff
@@ -0,0 +1,134 @@
+
+        SCHEMA_FIELDS_TO_CONVERT = schemaFieldMapBuilder.build();
+    }
+
+    /**
+     * Helper method to get the schema key for a DDB health data record
+     *
+     * @param record
+     *         DDB Item representing a health data record
+     * @return the schema key associated with that record
+     */
+    public static UploadSchemaKey getSchemaKeyForRecord(Item record) {
+        String studyId = record.getString("studyId");
+        String schemaId = record.getString("schemaId");
+        int schemaRev = record.getInt("schemaRevision");
+        return new UploadSchemaKey.Builder().withStudyId(studyId).withSchemaId(schemaId).withRevision(schemaRev)
+                .build();
+    }
+
+    /**
+     * Helper method to extract and sanitize a DDB string value, given a Dynamo DB item and a field name.
+     *
+     * @param item
+     *         Dynamo DB item
+     * @param fieldName
+     *         DDB field name
+     * @param maxLength
+     *         max length of the column
+     * @param recordId
+     *         record ID, for logging purposes
+     * @return sanitized DDB string value
+     */
+    public static String sanitizeDdbValue(Item item, String fieldName, int maxLength, String recordId) {
+        String value = item.getString(fieldName);
+        return sanitizeString(value, maxLength, recordId);
+    }
+
+    /**
+     * Helper method to extract and sanitize a JSON string value, given a JsonNode and a field name.
+     *
+     * @param node
+     *         JsonNode to extract the value from
+     * @param fieldName
+     *         JSON field name
+     * @param maxLength
+     *         max length of the column
+     * @param recordId
+     *         record ID, for logging purposes
+     * @return sanitized JSON string value
+     */
+    public static String sanitizeJsonValue(JsonNode node, String fieldName, int maxLength, String recordId) {
+        if (!node.hasNonNull(fieldName)) {
+            return null;
+        }
+        return sanitizeString(node.get(fieldName).textValue(), maxLength, recordId);
+    }
+
+    /**
+     * Sanitizes the given string to make it acceptable for a TSV to upload to Synapse. This involves removing
+     * TSV-unfriendly chars (newlines, carriage returns, and tabs) and truncating columns that are too wide. This
+     * will log an error if truncation happens, so this can be detected post-run.
+     *
+     * @param in
+     *         value to sanitize
+     * @param maxLength
+     *         max length of the column
+     * @param recordId
+     *         record ID, for logging purposes
+     * @return sanitized string
+     */
+    public static String sanitizeString(String in, int maxLength, String recordId) {
+        if (in == null) {
+            return null;
+        }
+
+        // First, remove tabs and newlines and carriage returns. This is needed to serialize strings into TSVs.
+        in = in.replaceAll("[\n\r\t]+", " ");
```

Comment:
We should do this for all names, identifiers, labels, etc. that we collect through the API. Also remove markup.